.github/workflows/agw.main.yml

```diff
@@ -40,5 +41,6 @@ jobs:
         run: pip install tox
       - name: Run tests using tox
         run: |
+          sudo apt install libsystemd-dev
           set -eux
           tox -e unit
```

Comment:
I see the pipeline runs fine but when I try to run unit tests I get the following error:

```
ImportError while importing test module '/home/guillaume/PycharmProjects/magma-access-gateway-snap/tests/unit/test_unit.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.9/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/unit/test_unit.py:9: in <module>
    from magma_access_gateway_installer.agw_installation_errors import (
magma_access_gateway_installer/__init__.py:5: in <module>
    from .agw_installer_ubuntu import AGWInstallerUbuntu
magma_access_gateway_installer/agw_installer_ubuntu.py:16: in <module>
    import netifaces  # type: ignore[import]
E   ModuleNotFoundError: No module named 'netifaces'
```

Reply:
Ahh I found the issue, it was with my Python version. I was running 3.9 instead of 3.8. Is there a way in the metadata.yaml file to specify that Python code has to be run with 3.8?